service/src/main/java/org/jboss/sbomer/service/feature/sbom/errata/event/release/AdvisoryReleaseEvent.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2023 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.sbomer.service.feature.sbom.errata.event.release;
+
+import java.util.List;
+import java.util.Map;
+
+import org.jboss.sbomer.core.dto.v1beta1.V1Beta1RequestRecord;
+import org.jboss.sbomer.service.feature.sbom.errata.dto.Errata;
+import org.jboss.sbomer.service.feature.sbom.errata.dto.ErrataBuildList.BuildItem;
+import org.jboss.sbomer.service.feature.sbom.errata.dto.ErrataBuildList.ProductVersionEntry;
+import org.jboss.sbomer.service.feature.sbom.model.SbomGenerationRequest;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder(setterPrefix = "with")
+public class AdvisoryReleaseEvent {
+
+    final Errata erratum;
+    final V1Beta1RequestRecord latestAdvisoryManifestsRecord;
```

Comment:
I was surprised to see such a record pas part of the event. I think I would prefer to have a "lightweight" event that would contain just the request identifier and on the handling side it would be queried, if necessary. 

Of course, one could argue that since we already have why not include it, but I think this is too specific to the API. My response would be  that we are not touching the API here at all. This is why I was surprised and I would probably decouple it from the API result object.

@vibe13 What are your thoughts on this?

Reply:
@goldmann you have the same thougths that crossed my mind and you have the same explanation of it. since we have those object why not using them? I reckon that we are passing a big event object indeed. I can definitely make it more lightweight and just pass around the mandatory information, then retrieve once again the corollary information by querying again the DB.

Reply:
That would be my preference, but let's act reasonably considering the time. Let's approve this one as it is and file a RFE to fix this. I'm happy to schedule it for 2025.